content/news/2023.pml

```diff
@@ -54,6 +54,22 @@ Older News: <%=reflist%>
 </div>
 -->
 
+<!----------------------------------------------------------->
+<div class="news-item" id="galois-joins-foundation">
+    29 May 2023: Galois joins the seL4 Foundation
+</div>
+<div>
+  <a href = "https://galois.com/">
+    <img src="/Foundation/Membership/LOGOS/galois-logo.svg"
+    style="width: 15%; float:right"
+    alt="Galois logo" />
+  </a>
+  <p>
+    Galois recently acquired Adventium Labs, long term members of the seL4 Foundation. Galois develops technology to guarantee the trustworthiness of systems where failure is unacceptable. They apply cutting edge computer science and mathematics to advance the state of the art in software and hardware trustworthiness. The seL4 Foundation welcomes its latest member and looks forward to our continuing collaboration.
```

Comment:
We are pleased to have Galois now part of the seL4 Foundation, following its acquisition of Adventium labs, which has been a member since 2020 and user of seL4 technologies for years before that. Galois develops [...]. The seL4 Foundation looks forward to our continuing collaboration."